packages/react-reconciler/src/ReactFiberCommitWork.js

```diff
@@ -613,9 +719,74 @@ function commitLifeCycles(
               finishedWork.actualDuration,
               finishedWork.treeBaseDuration,
               finishedWork.actualStartTime,
-              getCommitTime(),
+              commitTime,
+            );
+          }
+        }
+
+        if (typeof onCommit === 'function') {
+          if (enableSchedulerTracing) {
+            onCommit(
+              finishedWork.memoizedProps.id,
+              current === null ? 'mount' : 'update',
+              effectDuration,
+              commitTime,
+              finishedRoot.memoizedInteractions,
             );
+          } else {
+            onCommit(
+              finishedWork.memoizedProps.id,
+              current === null ? 'mount' : 'update',
+              effectDuration,
+              commitTime,
+            );
+          }
+        }
+
+        // In the common case, we don't call onPostCommit() here.
+        // Typically, passive effects haven't been measured yet,
+        // so any durations we have recorded will be for the previous commit's effects.
+        // However if we have recorded passive effect durations,
+        // it means cleanup functions were run as part of an unmount.
+        // In that event we should to call onPostCommit() and bubble durations.
+        //
+        // TODO (bvaughn, acdlite) We plan to refactor passive effect cleanup timing as part of the CM umbrella.
+        // Hopefully we can remove this codepath after that refactor?
```

Comment:
Do we have a task that I can link this TODO to? I didn't see one in the CM umbrella doc.